src/pudl/metadata/fields.py

```diff
@@ -60,6 +60,150 @@
+        "type": "number",
+    },
+    "op_gas_consumption_by_other": {
+        "description": (
+            "Volume of natural gas used in other company's operations"
+            "(14.73 psia and 60° Fahrenheit)"
+        ),
+        "unit": "Mcf",
+        "type": "number",
+    },
+    "operations_volume_underground_storage_injections": {
+        "description": (
+            "Underground storage injections volume (14.73 psia and 60° Fahrenheit)"
+        ),
+        "unit": "Mcf",
+        "type": "number",
+    },
+    "operations_volume_lng_storage_injections": {
+        "description": (
+            "Liquified natural gas (LNG) storage injections volume (14.73 psia and 60° Fahrenheit)"
+        ),
+        "unit": "Mcf",
+        "type": "number",
+    },
+    "lease_use_volume": {
+        "description": (
+            "Lease use volume (producers only) (14.73 psia and 60° Fahrenheit)"
+        ),
+        "unit": "Mcf",
+        "type": "number",
+    },
+    "returns_for_repress_reinjection_volume": {
+        "description": (
+            "Volume of gas returned to oil or gas reservoirs, used for repressuring, reinjection "
+            "(producers only) (14.73 psia and 60° Fahrenheit)"
+        ),
+        "unit": "Mcf",
+        "type": "number",
+    },
+    "losses_from_leaks_volume": {
+        "description": (
+            "Losses from leaks, damage, accidents, migration and/or blow down"
+            "(14.73 psia and 60° Fahrenheit)"
+        ),
+        "unit": "Mcf",
+        "type": "number",
+    },
+    "disposition_to_distribution_companies_volume": {
+        "description": (
+            "Disposition to distribution companies within the report state"
+            "(14.73 psia and 60° Fahrenheit)"
+        ),
+        "unit": "Mcf",
+        "type": "number",
+    },
+    "disposition_to_other_pipelines_volume": {
+        "description": (
+            "Dispositiont to other pipelines within the report state"
+            "(14.73 psia and 60° Fahrenheit)"
+        ),
+        "unit": "Mcf",
+        "type": "number",
+    },
+    "disposition_to_storage_operators_volume": {
+        "description": (
+            "Dispositiont to storage operators within the report state"
+            "(14.73 psia and 60° Fahrenheit)"
+        ),
+        "unit": "Mcf",
+        "type": "number",
+    },
+    "total_disposition_volume": {
+        "description": (
+            "Total disposition within the report state(14.73 psia and 60° Fahrenheit)"
+        ),
+        "unit": "Mcf",
+        "type": "number",
+    },
+    "unaccounted_for": {
+        "description": (
+            "Difference between gas supply (+) and disposition (-)"
+            "(14.73 psia and 60° Fahrenheit)"
+        ),
+        "unit": "Mcf",
+        "type": "number",
+    },
+    "out_of_state_deliveries_volume": {
+        "description": (
+            "Total volume of operator's out of state deliveries"
+            "(14.73 psia and 60° Fahrenheit)"
```

Comment:
```suggestion
            "Total volume of the operator's deliveries across or to state lines or U.S. borders. This has been summed from the detailed data reported by each company on Line 14.0 of the original form in order to preserve the primary key of the table. Reference conditions for measurement are 14.73 psia and 60° Fahrenheit."
```

See my main comments re: what to do with the set of fields reported in 14.0. When we add the detailed table, we should reference it explicitly in this description.